pysmt/smtlib/parser/parser.py

```diff
@@ -637,7 +636,10 @@ def atom(self, token, mgr):
                 res = mgr.BV(value, width)
             elif token[0] == '"':
                 # String constant
-                res = mgr.String(token.replace('"',''))
+                val = token[1:-1]
+                val = val.replace('""', '"')
+                val = val.decode('string-escape')
```

Comment:
This I'm not entirely sure about, this all depends on whether or not support for '\xFF' style escapes, etc. is desirable. 
Please let me know!

Reply:
This `decode()` is not availabe in python3 (see: https://stackoverflow.com/questions/14820429/how-do-i-decodestring-escape-in-python3?utm_medium=organic&utm_source=google_rich_qa&utm_campaign=google_rich_qa) and makes the tests fail on our CI: https://travis-ci.org/pysmt/pysmt/builds/385986747?utm_source=github_status&utm_medium=notification